frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -121,6 +115,7 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
           <LogsTabStatus
             loaded={loaded}
             error={error}
+            isLogsAvailable={podContainers.length === 1 && !logs}
```

Comment:
If you change the name to `isLogsAvailable` from `isLogsNotAvailable`, then you should pass the correct boolean to reflect the change.

Reply:
Agreed. I just changed this. Thanks.